### PR TITLE
[CUBRIDQA-60] add run_jdbc for daily regression

### PR DESCRIPTION
**Pull request by supler**
@fanzaiqiang 

add run_jdbc.sh for daily regression

### CODE REVIEW

File: CTP/common/ext/run_jdbc.sh [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+    ini.sh -s common -u "main.testing.category=${BUILD_SCENARIOS}" $ctp_jdbc_test_conf
+
+    #STEP 5: EXECUTE
+    ctp.sh jdbc -c $ctp_jdbc_test_conf | tee $tmplog
+
+    #STEP 6: BACKUP LOGS
+    cd $CTP_HOME/result/jdbc
+    current_id=`cat $CTP_HOME/result/jdbc/current_runtime_logs/current_task_id | tail -n 1`
+    timestamp=`date +'%Y.%m.%d_%H.%M.%S'`
+    backup_file=`jdbc_result_${{build_id}_${current_id}_${timestamp}`
```
Comment (fanzaiqiang) [inline CTP/common/ext/run_jdbc.sh]: Please fix '${{build_id}'

File: CTP/common/ext/run_jdbc.sh [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+    mkdir -p ${backup_file} 
+    cp -rf current_runtime_logs $backup_file
```
Comment (fanzaiqiang) [inline CTP/common/ext/run_jdbc.sh]: It's better not to copy. Just apply tar in place.

File: CTP/common/ext/run_jdbc.sh [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+	ctp_jdbc_test_conf=${CTP_HOME}/conf/jdbc_template.conf
+    else
+	ctp_jdbc_test_conf=${CTP_HOME}/conf/jdbc.conf
+    fi
+
+    if [ "$BUILD_TYPE" != 'general' -a "$BUILD_TYPE" != 'debug' ]; then
+	echo ""
+        echo "The current CTP does not support $BUILD_TYPE testing!"
+	exit 
+    elif [ "$BUILD_SCENARIOS" != "jdbc" -a "$BUILD_SCENARIOS" == "jdbc_debug" ]; then
```
Comment (fanzaiqiang) [inline CTP/common/ext/run_jdbc.sh]: as is:
elif [ "$BUILD_SCENARIOS" != "jdbc" -a "$BUILD_SCENARIOS" == "jdbc_debug" ]; then

to be:
1. elif -> if
2. 'jdbc_debug' is acceptable one.

**Comment by supler** [comment]
correct condition and typo

**Comment by fanzaiqiang** [comment]
@supler 

Go ahead to commit it. :smiley:

**Pull request merged**